pkg/deployment/deployment.go

```diff
@@ -326,6 +327,35 @@ func addServiceExtraMounts(
 
 		aeeSpec.ExtraMounts = append(aeeSpec.ExtraMounts, volMounts)
 	}
+
+	// Add mount for TLS certs
+	if service.Spec.HasTLSCerts {
+		volMounts := storage.VolMounts{}
+		secretName := nodeSet.Name + "-" + service.Name + "-certs"
+		sec := &corev1.Secret{}
+		err := client.Get(ctx, types.NamespacedName{Name: secretName, Namespace: service.Namespace}, sec)
```

Comment:
This will fails for services other than nova and libvirt as the cert secret is only created for those services but each DataPlaneService can have HasTLSCerts: true configured.

Reply:
yeah -- I had simply put in nova and libvirt as examples.  But I've rewritten things so that any service that has hasTLSCerts: True will have certs created for them.  See the next version.

Reply:
see the changes in controllers/openstackdataplanenodeset_controller.go

Reply:
fyi, there is now a tls pkg https://github.com/openstack-k8s-operators/lib-common/blob/main/modules/common/tls/tls.go we use on the ctlplane side to mount the service/ca certs to the deployment, then using kolla_start to e.g. add the ca cert to the ca store. not sure if it can be used as is on the edpm side.